### PR TITLE
Feature/entries in source table

### PR DESCRIPTION
**Pull request by subinasr**
- Addresses #3986 , #3987 , #3989

## Changes
 * Add entries in sources table
 * Add entry delete functionality
 
## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

### CODE REVIEW

**Review by samshara — DISMISSED** [review]

**Review by tnagorra — COMMENTED** [review]
IDK if this PR is complete

**Comment by subinasr** [comment]
> IDK if this PR is complete

**Comment by subinasr** [comment]
@tnagorra Can you tell me why you think it's incomplete?

File: app/views/Project/Tagging/Sources/components/EditableEntry/index.tsx [hunk 1]
```diff
@@ -133,6 +146,32 @@ function EditableEntry(props: Props) {
...
+                );
+                onEntryDataChange();
+            },
+            onError: (gqlError) => {
+                alert.show(
+                    gqlError.message,
+                    { variant: 'error' },
+                );
+                // eslint-disable-next-line no-console
+                console.error(gqlError);
```
Comment (AdityaKhatri) [inline app/views/Project/Tagging/Sources/components/EditableEntry/index.tsx]: This is not required. Remove from both mutations

File: app/views/Project/Tagging/Sources/SourcesGrid/EntryCard/index.tsx [hunk 2]
```diff
@@ -62,10 +76,44 @@ function EntryCard(props: Props) {
...
+    );
+
+    const handleEntryDeleteClick = useCallback(() => {
+        deleteEntry({
+            variables: {
+                projectId,
+                entryId: entry.id,
+            },
+        });
+    }, [projectId, entry.id]);
```
Comment (samshara) [inline app/views/Project/Tagging/Sources/SourcesGrid/EntryCard/index.tsx]: add ```deleteEntry``` as dependency to useCallback hook

File: app/views/Project/Tagging/Sources/components/EditableEntry/index.tsx [hunk 3]
```diff
@@ -248,6 +285,31 @@ function EditableEntry(props: Props) {
...
     );
 
+    const handleEntryDeleteButtonClick = useCallback(() => {
+        deleteEntry({
+            variables: {
+                projectId,
+                entryId: entry.id,
+            },
+        });
+    }, [projectId, entry.id]);
```
Comment (samshara) [inline app/views/Project/Tagging/Sources/components/EditableEntry/index.tsx]: add ```deleteEntry``` to useCallback hook dependency

File: app/views/Project/Tagging/Sources/components/EditableEntry/index.tsx [hunk 4]
```diff
@@ -248,6 +285,31 @@ function EditableEntry(props: Props) {
...
+
+    const entryDeleteButton = (
+        <ConfirmButton
+            name={undefined}
+            variant="secondary"
+            onClick={handleEntryDeleteButtonClick}
+            message="Are you sure you want to delete the entry?"
+            disabled={deleteEntryPending}
```
Comment (tnagorra) [inline app/views/Project/Tagging/Sources/components/EditableEntry/index.tsx]: Should also be disabled on edit mode.
Let's disable all other actions on edit mode

**Review by tnagorra — CHANGES_REQUESTED** [review]

**Pull request merged**